html/elements/iframe.json

```
@@ -1036,7 +1036,8 @@
                 "version_added": "43"
               },
               "safari": {
-                "version_added": null
+                "version_added": "11.1",
+                "notes": "Not initially available in 11.1, but added in sub-version 13605.1.33.1.2. Also backported to version 10.13.4 on MacOS High Sierra."
```

After rereading the issue you linked to, I'm going to suggest dropping the macOS version information, since it's really about which version of of macOS got the updated version of Safari, not the Safari version itself, which is what we typically record for desktop Safari in BCD. Does that sound OK to you?

```suggestion
                "notes": "Not initially available in 11.1, but added in sub-version 13605.1.33.1.2."
```